charts/celeborn/templates/worker/statefulset.yaml

```
@@ -56,6 +56,9 @@ spec:
       {{- end }}
       {{- if $paths }}
       initContainers:
+        {{- if .Values.worker.extraInitContainers }}
+          {{- tpl (toYaml .Values.worker.extraInitContainers) . | nindent 6 }}
+        {{- end }}
```

1.  Can be simplifed as:
```suggestion
      {{- with .Values.worker.extraInitContainers }}
      {{- toYaml . | nindent 6 }}
      {{- end }}
```
2. The if statement `{{- if $paths }}` should be moved after extraInitContainers, otherwise the `extraInitContainers` might not be rendered due to empty paths list.